### DISCUSSION TITLE
Goaltracking issue

### DISCUSSION THREAD

**Discussion by webtail** [plausible/analytics#2275] [Anything Else]
Hi,

These are the goal settings i've been using in order to catch a sale: 

![01](https://user-images.githubusercontent.com/43131449/192525140-751bdfb5-0be5-4178-a51f-35434a046b90.jpg)

But it's doesn't work although sales are coming in and can be found in the pages report

![02](https://user-images.githubusercontent.com/43131449/192525296-bd1cbe67-85f2-436c-add4-b1420434c137.jpg)

No idea what i'm doing wrong?

Thxs

**Comment by metmarkosaric** [comment]
hi @webtail! you have a trailing slash so please use double asterisks (**). you can read more here https://plausible.io/docs/pageview-goals#pageview-goals-support-wildcards

**Reply by webtail** [reply]
Thxs @metmarkosaric . Problem solved :)
